src/com/animation.js

```diff
@@ -0,0 +1,204 @@
+		/**
+		 * Set the direction of the animation (-1 to play in reverse)
+		 * @param {Integer} direction the wanted direction of the animation
+		 */
+		var setDirection = function (direction) {
+			renderer.setDirection(direction || 1); // 1 is normal direction.
+		};
+
+		/**
+		 * Play only parts of an animation
+		 * @param {Array} segments range of frame to play [fromFrame, toFrame]
+		 * @param {*} force Force the segment to play now
+		 */
+		var playSegments = function (segments, force) {
+			renderer.playSegments(segments, force);
+		};
+
+		/**
+		 * Plays the animation.
+		 * Please refer to the documentation if this method is not clear enough :)
+		 */
+		var play = function () {
+			if (!isPlaying) {
+				renderer.play();
+				isPlaying = true;
+			}
+		};
+
+		/**
+		 * Pause the animation
+		 */
+		var pause = function () {
+			if (!!isPlaying) {
+				renderer.pause();
+				isPlaying = false;
+			}
+		};
+
+		/**
+		 * Stop the animation
+		 */
+		var stop = function () {
+			renderer.stop();
+			isPlaying = false;
+		};
+
+		/**
+		 * Destroy the world, and yeah, the animation.
+		 */
+		var destroy = function () {
+			renderer.destroy();
+			renderer = null;
+		};
+
+		/**
+		 * Compute the wanted option for a specific animation with the default values
+		 * and the given config in the data-attr.
+		 */
+		var computeOptions = function () {
+			var opts = {};
+			var dataAttrPattern = new RegExp('^animation');
+			opts = _.reduce(animation.data(), function (memo, value, key) {
+				if (dataAttrPattern.test(key)) {
+					if (_.isObject(value)) {
+						return memo;
+					}
+					var parsedKey = key.replace(dataAttrPattern, '');
+					var validKey = '';
+					if (!!parsedKey && !!parsedKey[0]) {
+						validKey = parsedKey[0].toLowerCase();
+						if (parsedKey.length >= 2) {
+							validKey += parsedKey.substr(1);
+						}
+						memo[validKey] = value;
+					}
+				}
+				return memo;
+			}, {});
+			return _.assign({
+				wrapper: animation.get(0)
+			}, defaults, opts);
+		};
+
+		/**
+		 * Render the animation
+		 */
+		var render = function () {
+			renderer = window.bodymovin.loadAnimation(computeOptions());
+			isPlaying = true;
+		};
+
+		/**
+		 * Just to be sure the library is loaded before making any calls.
+		 */
+		var bm = function () {
+			return !!window.bodymovin;
+		};
+
+		/**
+		 * Check if the animation is inited (It may not play but can be inited).
+		 */
+		var inited = function () {
+			return !!renderer;
+		};
+
+		/**
+		 * Check if the animation is playing
+		 */
+		var playing = function () {
+			return inited() && isPlaying;
+		};
+
+		/**
+		 * Get the jQuery element that contains the animation.
+		 */
+		var get = function () {
+			return animation;
+		};
+
+		/**
+		 * Set the ctn of the animation and call render if everything is ready
+		 * @param {jQuery Element} scope the container of the animation
+		 */
+		var init = function (scope) {
+			animation = scope;
+
+			App.loaded(bm, function () {
```

Comment:
The anonymous function can be removed.

```suggestion
			App.loaded(bm, render);
```

Reply:
ping @fhamon